packages/create-plugin/templates/common/.config/jest.config.js

```diff
@@ -5,6 +5,8 @@
  */
 
 const path = require('path');
+// Jest will throw errors if it tries to load an es module with it being transformed first
+const esModules = ['ol', 'react-colorful'].join('|');
```

Comment:
Maybe it would make sense to update the comment and the name of the variable to reflect more what problems one can solve with adding package names here and when it should be used? Probably it would also make sense to export it so plugin devs can extend it in their own config more easily?

```suggestion
// Just an idea, maybe we can find a better name
const esModuleNpmPackages = ['ol', 'react-colorful'].join('|');
```